modules/c++/math/tests/complexBenchmark2.cpp

```diff
@@ -0,0 +1,160 @@
+/* Users guide
```

Comment:
Can't these two benchmarks be combined into one where you just optionally loop N times?